### PR TITLE
Shallow render

### PR DESCRIPTION
**Pull request by albertfdp**

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -143,6 +144,14 @@ const unexpectedEnzyme = {
       }
     });
 
+    childExpect.exportType({
+      name: 'ShallowWrapper',
+      base: 'ReactWrapper',
```
Comment (sunesimonsen) [inline src/index.js]: Have you checked that the shallow wrapper is a clean subset of the full render?

**Review by sunesimonsen — APPROVED** [review]
The only thing I'm worried about is the places where we use htmllike. But let's see if anything weird arises.

File: src/index.js (hunk 1 above)
Comment (albertfdp) [reply src/index.js]: Do you mean for the assertions we support in `ReactWrapper`? Would it be better if we did that manually when adding assertions? `<ReactWrapper|ShallowWrapper>`?

Comment (sunesimonsen) [reply src/index.js]: We could have one Wrapper base and two types. I think this inheritance will bite us. where you want to special case ShallowWrapper.

Comment (albertfdp) [reply src/index.js]: Sounds sane. I'll fix that later.

Comment (sunesimonsen) [reply src/index.js]: Probably better to call the base EnzymeWrapper

File: src/index.js [hunk 2]
```diff
@@ -166,12 +167,21 @@ const unexpectedEnzyme = {
...
+      '<ReactWrapper> [not] to be checked',
       (expect, reactWrapper) => {
         expect(reactWrapper.instance().checked, '[not] to be true');
       }
     );
 
+    childExpect.exportAssertion(
+      '<ShallowWrapper> [not] to be checked',
+      (expect, shallowWrapper) => {
+        const element = $(shallowWrapper.html());
```
Comment (albertfdp) [inline src/index.js]: @sunesimonsen what do you think about using `cheerio`? I'm taking a similar approach as https://github.com/producthunt/chai-enzyme/blob/master/src/ShallowTestWrapper.js#L80

It seems that it's a common practise when using enzyme, as it has built-in support http://airbnb.io/enzyme/docs/api/render.html

Comment (sunesimonsen) [reply src/index.js]: I sounds slow, so I think we should only do it in cases where the normal approach doesn't work.

**Review by sunesimonsen — APPROVED** [review]
👍

**Pull request merged**